src/DataStructures/Transpose.hpp

```diff
@@ -30,7 +30,7 @@ void transpose_impl(double* matrix_transpose, const double* matrix,
 template <typename T>
 void raw_transpose(const gsl::not_null<T*> result, const T* const data,
                    const size_t chunk_size, const size_t number_of_chunks) {
-  if constexpr (std::is_same_v<double, T>) {
+  if constexpr (false) {
```

Comment:
[Optional] In addition to the `constexpr false`, keep the current line, just commented out with a comment above saying that it causes seg faults for some edge cases and needs to be tested more.